src/test/java/com/feedzai/commons/sql/abstraction/engine/impl/abs/AbstractEngineSchemaTest.java

```diff
@@ -639,9 +741,65 @@ protected EntityEntry createSpecialValueEntry(final Object columnValue) {
                 .build();
     }
 
-    protected void defineUDFGetOne(DatabaseEngine engine) throws DatabaseEngineException {
+    /**
+     * Defines the UDF "GetOne" in the database engine.
+     *
+     * @param engine The database engine.
+     * @throws DatabaseEngineException If anything goes wrong creating the UDF.
+     */
+    protected void defineUDFGetOne(final DatabaseEngine engine) throws DatabaseEngineException {
     }
 
-    protected void defineUDFTimesTwo(DatabaseEngine engine) throws DatabaseEngineException {
+    /**
+     * Defines the UDF "TimesTwo" in the database engine.
+     *
+     * This UDF is supposed to be created in the {@link #getTestSchema() test schema} (if the database supports it).
+     *
+     * @param engine The database engine.
+     * @throws DatabaseEngineException If anything goes wrong creating the UDF.
+     */
+    protected void defineUDFTimesTwo(final DatabaseEngine engine) throws DatabaseEngineException {
     }
+
+    /**
+     * Creates the {@link #getTestSchema() test schema} in the database, dropping it first if necessary.
+     *
+     * @throws DatabaseEngineException  If anything goes wrong creating the schema.
+     * @throws DatabaseFactoryException If anything goes wrong connecting to the database.
+     * @since 2.1.13
+     */
+    public void dropCreateTestSchema() throws DatabaseEngineException, DatabaseFactoryException {
+        try (final DatabaseEngine engine = DatabaseFactory.getConnection(properties)) {
+            try {
+                dropSchema(engine, getTestSchema());
+            } catch (DatabaseEngineException e) {
+                // do nothing
```

Comment:
Why not just remove the try-catch here and let the Exception being thrown upwards to the other methods?